server/src/main/java/com/openelements/benchscape/server/store/endpoints/BenchmarkEnvironmentMetadataEndpoint.java

```diff
@@ -305,11 +320,49 @@ public List<String> getAllGitBranches(@RequestParam final String gitOrigin) {
                 .toList();
     }
 
+    /**
+     * Get a list of all human-readable system memory descriptions that are currently part of environments (for the current
+     * tenant - see #{@link EnvironmentService}).
+     *
+     * @return all human-readable system memory descriptions
+     */
     @GetMapping("/systemMemoryReadable")
     public List<String> getAllSystemMemoryReadable() {
```

Comment:
With the new `SystemMemory` record data type I would live to just return a list of them (`List<SystemMemory>`).